roles/ci_gen_kustomize_values/molecule/default/prepare.yml

```diff
@@ -56,3 +56,8 @@
       ansible.builtin.git:  # noqa: latest[git]
         repo: "https://github.com/openstack-k8s-operators/architecture"
         dest: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/architecture"
+
+- name: Download tools for later testing and validations
+  ansible.builtin.import_playbook: >-
```

Comment:
Why not using this same notation? https://github.com/openstack-k8s-operators/ci-framework/pull/1073/files#diff-7f9fbb85b66eb3b26a6abc6a28034e080eb0c8e7629216f1659be3691c612913R58
Or the one that uses something like this? `zuul.projects['github.com/openstack-k8s-operators/install_yamls'].src_dir`

Reply:
that may have been a rest of the tests (and failures) I tried to correct with the ansible-lint + "missing playbook".